Добавлена возможность использовать заранее скачанные дистрибутивы

Daabramov opened the pull request.

Daabramov committed:
feat: Добавлена возможность использовать готовые дистрибутивы

Daabramov committed:
Правка gitignore

nixel2007 commented on README.md:
```diff
@@ -42,6 +42,10 @@ $ cp .env.example .env
 
+## Как использовать готовые дистрибутивы
+
+Вы можете использовать готовые дистрибутивы платформы, для этого достаточно в папку `distr`. Скрипты сборки будут автоматически использовать их для билда образа.
```
просто космос

nixel2007 replied:
```suggestion
Вы можете использовать готовые дистрибутивы платформы, для этого достаточно разместить их в папке `distr`. Скрипты сборки будут автоматически использовать их для сборки образа.
```

nixel2007 committed:
Update README.md

nixel2007 commented on crs/Dockerfile:
```diff
@@ -10,11 +10,16 @@ COPY ./scripts/download_og.sh /download_og.sh
+  && sync; /download_og.sh; \
+  fi
+
+RUN rm -rf ${downloads}/*thin*.tar.gz \
```
очень не хотелось бы разбивать это на 2 разных слоя. это приведет к большому росту размера образа. лучше продублировать команды по удалению в двух RUN

Daabramov committed:
fix: Правки по ревью

- Объединение в 1 слой
- Прокинут 1545 порт для ras

Daabramov committed:
feat: Подддержка скачанного дистрибутива edt

- Добавлена возможность использовать уже скачанный дистрибутив едт
- При копировании дистрибутивов теперь удаляются лишние файлы
- Команды использования дистрибутивов, их удаления и использования объеденены в 1 слой
